mage_ai/frontend/playwright.config.ts

```diff
@@ -10,36 +10,41 @@ import { defineConfig, devices } from '@playwright/test';
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  testDir: './tests',
-  /* Run tests in files in parallel */
-  fullyParallel: true,
-  /* Fail the build on CI if you accidentally left test.only in the source code. */
-  forbidOnly: !!process.env.CI,
-  /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
-  timeout: 60000,
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
-  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
-  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
-  use: {
-    /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: 'http://localhost:3000',
-
-    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on',
+  // This section is the same as in the playwright.config.ci.ts file.
+  expect: {
+    // Maximum time expect() should wait for the condition to be met.
+    timeout: 30000,
   },
-
-  /* Configure projects for major browsers */
+  // Fail the build on CI if you accidentally left test.only in the source code.
+  forbidOnly: !!process.env.CI,
+  // Run tests in files in parallel
+  fullyParallel: true,
+  // Configure projects for major browsers
   projects: [
     {
       name: 'chromium',
       use: { ...devices['Desktop Chrome'] },
     },
   ],
+  // Reporter to use. See https://playwright.dev/docs/test-reporters
+  reporter: 'html',
+  // Retry on CI only
+  retries: process.env.CI ? 2 : 0,
+  testDir: './tests',
+  // Time spent by the test function, fixtures, beforeEach and afterEach hooks
+  timeout: 75000,
```

Comment:
@edmondwinston Removed the `VISIBLE_TIMEOUT` constant cuz you can actually adjust it for all the tests in the playwright config.